### ISSUE TITLE
Disable LEUpdater, prevent LEProc stalling

### ISSUE THREAD

**Issue by dimechime** [xupefei/Locale-Emulator#395]
Hi, I am having some trouble getting LE to work on a computer where I do not have admin access. I have installed LE for just the current user, and tried launching a program. However, it seems like the LEUpdater process keeps getting killed because I don't have admin access (it keeps popping in and out of Task Manager), and so LEProc stalls and the program never launches. 
Is there anyway to get around this?

**Comment by xupefei** [comment]
I don't think it is the real cause... but anyway you can safely rename or delete LEUpdater.exe without any problem.
